TaskMarsCompetition/TestMarsCompetition/Utilities/TestDataCertificate.json

```diff
@@ -0,0 +1,282 @@
+﻿{
+  "LoginData": {
```

Comment:
so with with your test data it should be spilt up per test, each test should have their own json file